olive/passes/onnx/transformer_optimization.py

```diff
@@ -175,6 +174,13 @@ def _run_for_config(
                         run_config["hidden_size"] = model_config[hidden_size_name]
                         break
 
+        if run_config["model_type"] is None or run_config["model_type"] not in transformers_optimizer.MODEL_TYPES:
```

Comment:
Is the `None` check to make the logic clear? Since `None` would also fail the second condition.

Reply:
I explicitly check the None to make the logic clear.